src/zzz_od/auto_battle/auto_battle_agent_context.py

```
@@ -382,6 +395,21 @@ def check_agent_related(self, screen: MatLike, screenshot_time: float) -> None:
         finally:
             self._check_agent_lock.release()
 
+    def _should_force_check_all_agents(self, screen_agent_list: List[Tuple[Agent | None, str | None]]) -> bool:
+        """
+        当前处于战斗画面，但一个角色都识别不到时，下一次强制重新全量识别。
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)